src/containers/changeAccountEmail/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Alert, Button, Form, Input, Typography } from 'antd';
+import { ContentContainer } from '../../components';
+import ConfirmationMessage from '../../components/ConfirmationMessage';
+import { Link as RouterLink } from 'react-router-dom';
+import ProtectedApiClient from '../../api/protectedApiClient';
+import styled from 'styled-components';
+
+const { Title, Link } = Typography;
+
+const UpdateButton = styled(Button)`
+  margin-top: 20px;
+`;
+
+const ChangeAccountEmail: React.FC = () => {
+  const [loading, setIsLoading] = useState(false);
+  const [error, setError] = useState(false);
+  const [emailChanged, setEmailChanged] = useState(false);
+
+  const onFinishChangeEmail = async (values: any) => {
```

Comment:
We should create an interface for the form types, it will be the same interface as the request for the API so we can also reuse it there
```suggestion
  const onFinishChangeEmail = async (values: ChangeEmailRequest) => {
```

```
interface ChangeEmailRequest {
    newEmail: string;
    password: string;
}
```